python/yapf_format.py

```diff
@@ -33,7 +33,7 @@ def main():
 
     encoding = vim.eval('&encoding')
     buf = vim.current.buffer
-    unicode_buf = [unicode(s, encoding) for s in buf]
+    unicode_buf = [s for s in buf]
```

Comment:
@cuerty this line does nothing now. Have you tried using `buf` directly in place of `unicode_buf` below?

Also you can remove the encoding variable as it is unused